ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -143,34 +134,33 @@ Feature: Integration - Import Export
     # import from resources TODO
     Then import integration from relative file path "src/test/resources/integrations/Imported-integration-another-instance-export.zip"
 
-    # check that connections need credentials update
     When navigate to the "Connections" page
-    And check visibility of alert notification
-    # update connections credentials
-    And click on the "View" kebab menu button of "QE Salesforce"
-    And check visibility of "QE Salesforce" connection details
-    Then check visibility of alert notification
+    And click on the "Edit" kebab menu button of "QE Slack"
+    Then check visibility of "QE Slack" connection details
 
     When click on the "Edit" button
-    And fill in "QE Salesforce" connection details from connection edit page
+    And fill in "QE Slack" connection details from connection edit page
     And click on the "Validate" button
     Then check visibility of success notification
-    When click on the "Save" button
-    And navigate to the "Integrations" page
+    And click on the "Save" button
+
+    When navigate to the "Integrations" page
 
     #should be unpublished after import
-    When select the "Integration_import_export_test" integration
+    And select the "Integration_import_export_test" integration
     Then check visibility of "Stopped" integration status on Integration Detail page
 
-    When sleep for jenkins delay or "3" seconds
-    # start integration and wait for published state:
-    And start integration "Integration_import_export_test"
+    When click on the "Edit" button
+    And click on the "Publish" button
 
-    When sleep for jenkins delay or "3" seconds
+    And sleep for jenkins delay or "3" seconds
+
+    Then check visibility of "Integration_import_export_test" integration details
     And navigate to the "Integrations" page
+
+    And Integration "Integration_import_export_test" is present in integrations list
+    # wait for integration to get in active state
+    And sleep for "300000" ms
```

Comment:
Please, for all the 5 min. and longer waits, find more dynamic solution. The whole test suite execution time would rise by more than 26 minutes and tbh I believe there is more elegant way to handle this.